api/control/v1alpha1/types.go

```diff
@@ -252,6 +252,10 @@ type BindingSpec struct {
 
 	// `destinations` is a list of cluster-identifiers that the objects should be propagated to.
 	Destinations []Destination `json:"destinations,omitempty"`
+
+	// `statusCollectors` is a list of StatusCollectors name references that appear in the associated BindingPolicy
+	// and are applied to one or more of the objects in the `workload` field.
+	StatusCollectors []string `json:"statusCollectors,omitempty"`
 }
 
 // DownsyncObjectReferences defines the objects to be down-synced, grouping them by scope.
```

Comment:
Analogously to what was done for BindingPolicy, I would not pretend that the status collector reference is part of the object reference. These are two distinct concepts. Were they are paired, have a pair type and types for the two individual components.

Reply:
done